.gitignore

```diff
@@ -20,4 +24,4 @@ standalonedata/
 
 worldwind.js
 
-worldwind.min.js
```

Comment:
I'm wondering about the order, would it be better to group index.js with the worldwind js files? Also to keep consistent with the other folders, should test-results get a trailing slash?